src/subdomains/supporting/payment/services/transaction-helper.ts

```diff
@@ -428,18 +428,17 @@ export class TransactionHelper implements OnModuleInit {
     paymentMethodOut: PaymentMethod,
     user: User,
   ): Promise<{ kycLimit: number; defaultLimit: number }> {
+    const volume24h =
+      user?.userData.kycLevel < KycLevel.LEVEL_50
+        ? await this.getVolumeSince(Util.daysBefore(1), new Date(), [user])
```

Comment:
Maybe we should adapt like in amlCheck `await this.getVolumeSince(Util.daysBefore(1), Util.daysAfter(1), [user])` or do you think we don't need that because edge case and only for PaymentInfos? 🤔 
I think one day in future should be not need much more resources than currentDate or?

Reply:
The `Util.daysAfter` in AML check is based on `transaction.created` because a transaction can be quite "old". Here it's always the current day and I hope we'll never have future transactions 😆

Reply:
Hehe jap your right 😁